roles/ceph-infra/tasks/setup_ntp.yml

```diff
@@ -1,14 +1,16 @@
 ---
-- name: set ntp service and chrony daemon name for Debian family
+- name: set ntp service, chrony and timesyncd daemon name for Debian family
   set_fact:
     chrony_daemon_name: chrony
     ntp_service_name: ntp
+    timesyncd_service_name: systemd-timesyncd
```

Comment:
Why do we need this variable?
The service name is the same L6 and L13

Reply:
Not for a particular reason. Only for coherence with the other services and for a better readability/maintenance